Extensions/TweenBehavior/JsExtension.js

```diff
@@ -0,0 +1,524 @@
+        _("Add a tween animation for the object Y position."),
+        _(
+          "Tween the Y position of _PARAM0_ to _PARAM3_ with easing _PARAM4_ over _PARAM5_ms as _PARAM2_"
+        ),
+        _("Tween animation/Position"),
+        "JsPlatform/Extensions/take_screenshot24.png",
+        "JsPlatform/Extensions/take_screenshot32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .addParameter("expression", _("To Y"), "", false)
+      .addParameter("stringWithSelector", _("Easing"), "[\"linear\", \"easeInQuad\", \"easeOutQuad\", \"easeInOutQuad\", \"easeInCubic\", \"easeOutCubic\", \"easeInOutCubic\", \"easeInQuart\", \"easeOutQuart\", \"easeInOutQuart\", \"easeInQuint\", \"easeOutQuint\", \"easeInOutQuint\", \"easeInSine\", \"easeOutSine\", \"easeInOutSine\", \"easeInExpo\", \"easeOutExpo\", \"easeInOutExpo\", \"easeInCirc\", \"easeOutCirc\", \"easeInOutCirc\", \"easeOutBounce\", \"easeInBack\", \"easeOutBack\", \"easeInOutBack\", \"elastic\", \"swingFromTo\", \"swingFrom\", \"swingTo\", \"bounce\", \"bouncePast\", \"easeFromTo\", \"easeFrom\", \"easeTo\"]", false)
+      .setDefaultValue("linear")
+      .addParameter("expression", _("Duration"), "", false)
+      .addParameter(
+        "yesorno",
+        _("Destroy this object when tween finishes"),
+        "",
+        false
+      )
+      .setDefaultValue("no")
+      .getCodeExtraInformation()
+      .setFunctionName("addObjectPositionYTween");
+
+    behavior
+      .addAction(
+        "AddObjectAngleTween",
+        _("Add object angle tween"),
+        _("Add a tween animation for the object angle."),
+        _(
+          "Tween the angle of _PARAM0_ to _PARAM3_° with easing _PARAM4_ over _PARAM5_ms as _PARAM2_"
+        ),
+        _("Tween animation/Angle"),
+        "JsPlatform/Extensions/take_screenshot24.png",
+        "JsPlatform/Extensions/take_screenshot32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .addParameter("expression", _("To angle (in degrees)"), "", false)
+      .addParameter("stringWithSelector", _("Easing"), "[\"linear\", \"easeInQuad\", \"easeOutQuad\", \"easeInOutQuad\", \"easeInCubic\", \"easeOutCubic\", \"easeInOutCubic\", \"easeInQuart\", \"easeOutQuart\", \"easeInOutQuart\", \"easeInQuint\", \"easeOutQuint\", \"easeInOutQuint\", \"easeInSine\", \"easeOutSine\", \"easeInOutSine\", \"easeInExpo\", \"easeOutExpo\", \"easeInOutExpo\", \"easeInCirc\", \"easeOutCirc\", \"easeInOutCirc\", \"easeOutBounce\", \"easeInBack\", \"easeOutBack\", \"easeInOutBack\", \"elastic\", \"swingFromTo\", \"swingFrom\", \"swingTo\", \"bounce\", \"bouncePast\", \"easeFromTo\", \"easeFrom\", \"easeTo\"]", false)
```

Comment:
We can remove this redundancy, making things easier to read and not a pain to update :) Create a variable for all of these:

```
//Earlier in the createExtension function, before starting the `addAction`s:
const easingChoices = JSON.stringify([
 'linear',
 'easeInQuad',
 ...
]);

// ...
// then simply do:
.addParameter("stringWithSelector", _("Easing"), easingChoices, false)

```